code/game/objects/structures/aliens.dm

```diff
@@ -82,23 +97,144 @@
 /obj/structure/alien/resin/wall/BlockSuperconductivity()
 	return TRUE
 
-/obj/structure/alien/resin/membrane
-	name = "resin membrane"
-	desc = "Resin just thin enough to let light pass through."
-	icon = 'icons/obj/smooth_structures/alien/resin_membrane.dmi'
-	icon_state = "resin_membrane-0"
-	base_icon_state = "resin_membrane"
-	opacity = FALSE
-	max_integrity = 160
-	resintype = "membrane"
+/*  Resin-Door
+	This borrs a bit of code from Mineral-Door, but is mostly its own thing due to not needing many of the latter's things instead of its own things */
```

Comment:
```suggestion
	This borrows a bit of code from Mineral-Door, but is mostly its own thing due to not needing many of the latter's things instead of its own things */
```

Reply:
Fixed in an upcoming commit 👍